components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -112,22 +112,24 @@ func (g *gpsrtk) connectAndParseSourceTable() error {
 }
 
 func (g *gpsrtk) getStream() (*rtcm3.Scanner, error) {
+	var streamSource io.Reader
+
 	if g.isVirtualBase {
 		g.logger.Debug("connecting to Virtual Reference Station")
 		err := g.getNtripFromVRS()
 		if err != nil {
 			return nil, err
 		}
-		reader := io.TeeReader(g.vrs.GetReaderWriter(), g.correctionWriter)
-		scanner := rtcm3.NewScanner(reader)
-		return &scanner, nil
-	}
-	g.logger.Debug("connecting to NTRIP stream........")
-	stream, err := g.ntripClient.GetStreamFromMountPoint(g.cancelCtx, g.logger)
-	if err != nil {
-		return nil, err
+		streamSource = g.vrs.GetReaderWriter()
+	} else {
+		g.logger.Debug("connecting to NTRIP stream........")
```

Comment:
Let's clarify what type of mount point we are connecting to! "connecting to static mount point "